src/backends/tensorflow/tensorflow_backend_tf.cc

```diff
@@ -632,6 +633,13 @@ TRTISTF_ModelCreateFromGraphDef(
         "model " + std::string(model_name) + " has an empty network");
   }
 
+  // Clear the device field from the graphdef so that TRTIS can control
```

Comment:
".. so that the default device setting below will control which GPU the graph will run on"

Reply:
done